tests/apidocs.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/sh
+#
+# After success script for apidocs on Travis CI.
+#
+# Apidoc doesn't work on php 7.2 because compatibility problems,
+# that's why this script is used on php 7.1 until the problem is solved
+#
+
+# find if we are in a valid branch to build docs
+if echo "$TRAVIS_BRANCH" | grep -q -P '^(master|develop|support/|release/)'; then
+    REGEX_BRANCH=true
+else
+    REGEX_BRANCH=false
```

Comment:
This variable has a bad name. Or tells of we must run or not apigen.

BUILD_DOC_API is a better name

Reply:
I left the same code name because it's the same conditional statement used on the after_success script.

Reply:
then my comment apply to the other script.